core/src/main/scala/org/apache/spark/util/SparkExitCode.scala

```diff
@@ -45,6 +45,9 @@ private[spark] object SparkExitCode {
       OutOfMemoryError. */
   val OOM = 52
 
+  /** Exit because the SparkContext is stopped. */
+  val SPARK_CONTEXT_STOPPED = 69
```

Comment:
`SparkContext` is an essential component/service of the Spark application, according to [1][2], 69 is a widely-used exit code with the typical meaning - Service unavailable: A service required to complete the task is unavailable.

[1] https://www.ditig.com/linux-exit-status-codes
[2] https://www.man7.org/linux/man-pages/man3/sysexits.h.3head.html